filebeat/docs/inputs/input-kafka.asciidoc

```diff
@@ -35,7 +55,7 @@ The `kafka` input supports the following configuration options plus the
 <<{beatname_lc}-input-{type}-common-options>> described later.
 
 [float]
-[[kafka-hosts]]
+[[hosts]]
```

Comment:
I think you'll need to rebase and use kafka-hosts here, or the doc build test will fail. Sorry didn't catch this before.